catalog/transformer/catalog-transformer-streaming-lib/src/main/java/org/codice/ddf/transformer/xml/streaming/lib/XmlInputTransformer.java

```diff
@@ -249,7 +242,7 @@ public void setOrganization(String organization) {
     }
 
     public MetacardType getMetacardType() {
-        return this.metacardType;
+        return defineMetacardType();
```

Comment:
Given how is this being used now, you could just inline `defineMetacardType` here and call `getMetacardType` instead, so the method name is more idiomatic with its usage.